.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+name: release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  package:
+    name: package and upload release assets
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - linux
+          - darwin
+          - windows
+        arch:
+          - amd64
+          - arm
+        exclude:
+          - os: darwin
+            arch: arm
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup-go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
```

Comment:
```
      matrix:
        os:
          - linux
          - darwin
          - windows
        arch:
          - amd64
          - arm64
          - arm
        exclude:
          - os: darwin
            arch: arm
          - os: windows
            arch: arm64
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 1
      - name: setup-go 1.16.4
        uses: actions/setup-go@v1
        with:
          go-version: 1.16.4
```
Maybe we can use this configuration. It  has been tested via nebula-console

Reply:
` go-version: 1.16.4`
We should require a lower version of Golang like 1.13

Reply:
It's just used when packaging releases in actions. When compiling it by source code, the needed golang version is still 1.13

Reply:
Fixed.